repository/filtering.go

```diff
@@ -9,14 +9,16 @@ import (
 

```

Comment:
These changes should not be in this branch now, am I right?

Reply:
i need it because i must test Match that use repository package. So i merged it from alpaka-alpachino's branch and generated mocks for that package